docs/static/releasenotes.asciidoc

```diff
@@ -36,6 +37,61 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-8-0]]
+=== Logstash 8.8.0 Release Notes
+
+[[features-8.8.0]]
+==== New features and enhancements
+
+* Negative acknowledgment of a batch. With this functionality a plugin can request the abortion a batch of event.
+  When used in combination with persisted queue it enables the reprocessing on Logstash restart https://github.com/elastic/logstash/pull/14940[#14940]
```

Comment:
```suggestion
* Adds functionality for negative acknowledgment when a batch contains errors or was not received.   
With this new functionality, a plugin can request that processing be stopped when a problem is detected in a batch.
When used in combination with a persistent queue, the batch is eligible for reprocessing when {ls} restarts.https://github.com/elastic/logstash/pull/14940[#14940]
```

Reply:
Above is a suggestion for rewording.  The wording suggestions are optional except for this: The word "abortion" must be replaced.

Reply:
Hi @karenzone I always appreciate a lot of your suggestion, and in particular this. 
Me neither like that word, but nothing more nice comes to me. 
The concept I want to express is: when a plugin rises that error not only it's asking to stop processing but also to repeat the processing of the same batch next it run. 
A stop on of a processing is "stop doing what you are doing and drop the data" while this kind of request is "stop what you are doing and permit (if the queue agrees on this) to reprocess the full batch from start (of the filters section)".